onnxruntime/contrib_ops/cpu/col2im.cc

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#include "core/providers/cpu/tensor/col2im.h"
+#include "core/providers/common.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+#define REGISTER_KERNEL_TYPED(T)                                                \
+  ONNX_OPERATOR_TYPED_KERNEL_EX(                                                \
+      Col2Im,                                                                   \
+      kMSDomain,                                                                \
+      1,                                                                        \
+      T,                                                                        \
+      kCpuExecutionProvider,                                                    \
+      KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<T>()), \
```

Comment:
built in clang formatter in the Studio would automatically align the right backslashes. You may want to turn it on automatically or on demand Ctrl+K+D

Reply:
Is there a specific issue here?  All backslashes are aligned already, after I executed `clang-format` for all modified cpp files

(I contribute to ort, pytorch, onnx, detectron2, etc so I try not to use automatic formatting due to conflicting format settings on my vscode.)

Reply:
Perhaps, it is a CF issue, but I am not seeing these as aligned. I thought that a single combination of keystrokes would do it.